ballerina/utils.bal

```diff
@@ -241,6 +241,9 @@ isolated function populateCreateObjectHeaders(map<string> requestHeaders, Object
         if (objectCreationHeaders?.expires != ()) {
             requestHeaders[EXPIRES] = <string>objectCreationHeaders?.expires;
         }
+        if (objectCreationHeaders?.contentType != ()) {
+            requestHeaders[CONTENT_TYPE] = <string>objectCreationHeaders?.contentType;
+        }
```

Comment:
Below is an improved version of the same implementation. 
```suggestion
        if objectCreationHeaders?.contentType is string {
            requestHeaders[CONTENT_TYPE] = <string>objectCreationHeaders?.contentType;
        }
```

But it'll okay to keep the current implementation, just to keep the consistency with the old code base. 

And the plan is to revisit the entire connector implementation, once we get an owner for this connector as a part of our connector reviving effort.